helpers/wrapper/contracts/Wrapper.sol

```diff
@@ -123,14 +128,16 @@ contract Wrapper {
       wethContract.withdraw(_makerAmount);
       // Transfer ether to the user.
       msg.sender.transfer(_makerAmount);
+    } else if (_makerToken != address(0)){
+      IERC20(_makerToken).transfer(msg.sender, _makerAmount);
     }
 
     // Ensure no WETH balance remains.
-    require(wethContract.balanceOf(address(this)) == 0,
+    require(wethContract.balanceOf(address(this)) == startingWETH,
       "WETH_BALANCE_REMAINING");
 
     // Ensure no ether balance remains.
-    require(address(this).balance == 0,
+    require(address(this).balance <= startingETH,
```

Comment:
Why is this one now an inequality, but line 136 is strict equality? Would it make sense to make 136 an inequality too?

Reply:
for some reason, with the integration tests, only <= seemed to work even though when i check the wrapper balance after the test strict equality should have held.need to dive into that a bit further, i think token arithmetic is fine it's specific to ether.

Reply:
I think we should make them BOTH `<=` though. It's slightly more risky from a security point of view doing strict equality in case something gets off by 0.000001. Thoughts?

Reply:
think based on slack, check is being removed. so it's a non-issue.